src/lsm/lsm_work_unit.c

```diff
@@ -429,6 +427,19 @@ __wt_lsm_checkpoint_chunk(WT_SESSION_IMPL *session,
 	if (ret != 0)
 		WT_ERR_MSG(session, ret, "LSM metadata write");
 
+	/*
+	 * Enable eviction on the live chunk so it doesn't block the cache.
+	 * Future reads should direct to the on-disk chunk anyway.
+	 */
+	btree = session->dhandle->handle;
+	if (btree->evict_disabled_open) {
+		btree->evict_disabled_open = false;
+		__wt_evict_file_exclusive_off(session);
+	}
```

Comment:
This is the real fix.  I know we've oscillated on when to allow eviction in LSM chunks, but there is a special case that avoids forced eviction when `btree->lsm_primary` is set, and it stays set after this, so I'm inclined to fix the test failures now and chase any fallout in LSM stability later.